src/PSRule/Pipeline/PipelineBuilder.cs

```diff
@@ -426,5 +426,10 @@ protected void AddVisitTargetObjectAction(VisitTargetObjectAction action)
             var previous = VisitTargetObject;
             VisitTargetObject = (targetObject) => action(targetObject, previous);
         }
+
+        protected static int LimitJsonIndentRange(int? jsonIndent)
+        {
+            return (jsonIndent.HasValue && 0 <= jsonIndent && jsonIndent <= 4) ? jsonIndent.Value : 0;
+        }
```

Comment:
Added this to normalize indentation if it falls outside the 0-4 bounds. Was thinking if throwing `ArgumentOutOfRange` exception, but not sure if thats necessary.

Reply:
I like your approach of normalization. Let's not throw an error if it's easy to understand the intent. Also, you could also make any value greater than 4 just 4.

Reply:
Thanks @BernieWhite. In that case I might as well rename to `NormalizeJsonIndentRange`.